examples/platform/silabs/efr32/rs911x/hal/sl_si91x_ncp_utility.c

```diff
@@ -160,52 +151,29 @@ sl_status_t spi_board_init()
  **********************************************************/
 sl_status_t sl_wfx_host_spi_cs_assert(void)

```

Comment:
isn't the function itself wrapped in `SL_SPICTRL_MUX` ?

Reply:
yes this is wrapped in SL_SPICTRL_MUX